source/common/http/async_client_impl.cc

```diff
@@ -20,6 +20,8 @@ const std::vector<std::reference_wrapper<const Router::RateLimitPolicyEntry>>
 const AsyncStreamImpl::NullHedgePolicy AsyncStreamImpl::RouteEntryImpl::hedge_policy_;
 const AsyncStreamImpl::NullRateLimitPolicy AsyncStreamImpl::RouteEntryImpl::rate_limit_policy_;
 const Router::InternalRedirectPolicyImpl AsyncStreamImpl::RouteEntryImpl::internal_redirect_policy_;
+const Router::PathMatcherSharedPtr AsyncStreamImpl::RouteEntryImpl::extension_path_matcher_;
```

Comment:
Please find every place in this entire PR where there is a new variable named "extension_" and remove the "extension_" prefix.

Reply:
removed